tests/tests_tangd_custom_port.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Test tangd_custom_port
+  hosts: all
+  vars:
+    nbde_server_port: 7500
+    nbde_server_firewall_zone: public
+  tasks:
+    - name: install with custom port and firewall zone
+      import_role:
+        name: linux-system-roles.nbde_server
+
+    - name: check if port is open
+      shell:
+        cmd: |-
+          set -euo pipefail
+          ss -tulpn | grep {{ nbde_server_port }} | awk -F' ' '{print $5}'
+      register: __open_ports_output
+      failed_when: __open_ports_output.stdout != "*:{{ nbde_server_port }}"
```

Comment:
@sergio-correia, @lessfoobar, @richm, how must this right-hand side be exactly matched?
The test playbook `tests_tangd_custom_port.yml` passes on `rhel-8` and `rhel-9`, but it fails on `rhel-7` as follows:
```
TASK [check if port is open] ***************************************************
task path: /home/nhosoi/linux-system-roles/nbde_server/tests/tests_tangd_custom_port.yml:12
Friday 23 September 2022  15:52:49 -0700 (0:00:00.651)       0:00:40.444 ****** 
[WARNING]: conditional statements should not include jinja2 templating
delimiters such as {{ }} or {% %}. Found: __open_ports_output.stdout != "*:{{
nbde_server_port }}"
fatal: [/home/nhosoi/.cache/linux-system-roles/rhel-7.qcow2]: FAILED! => {
    "changed": true,
    "cmd": "set -euo pipefail\nss -tulpn | grep 7500 | awk -F' ' '{print $5}'",
    "delta": "0:00:00.123563",
    "end": "2022-09-23 18:52:49.938979",
    "failed_when_result": true,
    "rc": 0,
    "start": "2022-09-23 18:52:49.815416"
}

STDOUT:
[::]:7500
```
Can we consider it is successful if the ports match?

Please note that `rhel-7` is a supported platform both as a control and a managed node.

Reply:
```suggestion
      failed_when: not __open_ports_output.stdout is search(':' ~ (nbde_server_port | str) ~ '$')
```